platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ImportMigrationConfigurationAdminContext.java

```diff
@@ -325,7 +324,7 @@ private ImportMigrationConfigurationAdminEntry proxy(ImportMigrationEntry entry)
 
   @SuppressWarnings(
       "PMD.UnusedFormalParameter" /* report parameter is required as this method is used as a functional interface */)
-  private void deleteUnexportedConfigurationsAfterCompletion(MigrationReport report) {
+  protected void deleteUnexportedConfigurationsAfterCompletion(MigrationReport report) {
```

Comment:
✏️ Since this is meant to be called by a class in the same package, it should be made package-private and not protected. Do add a comment as to which class is calling it.

Reply:
I do agree this should be package-private, but is it common to add a comment as to which class is calling it? Does this add value / are there other examples of this?

Reply:
It just make it simpler for the next guy to know that you didn't use package private by mistake.

Reply:
I think there are also static analysis tools that don't like that. I have other examples in this project of suppression with comments like these ... I think

Reply:
This will be addressed in #5354 in order to keep ports and commits consistent